Create ui-c8y-1021-73-2-use-custom-final-message-for-device-registration-via-est-8954-graft.md (#3305)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-73-2-use-custom-final-message-for-device-registration-via-est-8954-graft.md

---
date: ""

> title: Use custom final message for device registration via EST (#8954) [GRAFT][release/cd] (#9047)

Reviewer: ```suggestion
title: Improved confirmation message after device registration via EST
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63399
version: 1021.73.2
---

> Use custom final message for device registration via EST (#8954) [GRAFT][release/cd] (#9047)

Reviewer: ```suggestion
Previously, the final message displayed after completed device registration via EST was the same as in case of regular registation, which was confusing for the users. With this change, the displayed message is more relevant for devices registered via EST, i.e. that they can directly request the signed certificates and use them to authenticate to the platform without any further manual acceptance process.
```

> Previously, the final message displayed after completed device registration via EST was the same as in case of regular registation, which was confusing for the users. With this change, the displayed message is more relevant for devices registered via EST, i.e. that they can directly request the signed certificates and use them to authenticate to the platform without any further manual acceptance process.

Reviewer 2: ```suggestion
Previously, the message displayed after completion of the device registration via EST (Enrollment over Secure Transport protocol) was the same as in case of the regular device registration, which was confusing. With this change, the displayed message is specific for devices registered via EST, that is, they can directly request the signed certificates and use them to authenticate to the platform without any further manual acceptance process.
```

Reviewer 2: @pawel-rynarzewski-c8y What is EST? We should spell it out in the first occurrence.

Reviewer: @BeateRixen, it stands for "Enrollment over Secure Transport' protocol.

> Previously, the message displayed after completion of the device registration via EST (Enrollment over Secure Transport protocol) was the same as in case of the regular device registration, which was confusing. With this change, the displayed message is specific for devices registered via EST, that is, they can directly request the signed certificates and use them to authenticate to the platform without any further manual acceptance process.

Reviewer 2: ```suggestion
Previously, the confirmation message displayed after the device registration via EST (Enrollment over Secure Transport protocol) was the same as in case of the regular device registration, which was confusing. With this change, the displayed message is specific for devices registered via EST, that is, they can directly request the signed certificates and use them to authenticate to the platform without any further manual acceptance process.
```
